tide/master/resources/js/tide.js

```diff
@@ -1018,12 +1069,107 @@ function setScale() {
 
   var wallMargin = (window.innerWidth - (wallWidth * zoomScale)) / 2;
   var wall = $("#wall");
-  wall.css({transform: 'scale(' + zoomScale + ')'});
+  wall.css({ transform: 'scale(' + zoomScale + ')' });
```

Comment:
same here. If this is the new style, then please use it consistently (cf. lines 992 - 1008)